app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk

```diff
@@ -0,0 +1,69 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Email notifications - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK Pay
```

Comment:
see the [settings layout](https://github.com/alphagov/pay-selfservice/blob/db3fe0987cbc47ae5bfeac1e22f90f09ebe87145/app/views/simplified-account/settings/settings-layout.njk#L5) for how to use settingsPageTitle, some parts of the title are included for you (like `- GOV.UK Pay`)